doc/source/main_install.rst

```diff
@@ -117,26 +117,10 @@ e.g., ``~/.local/bin``.
 Build from source
 ~~~~~~~~~~~~~~~~~
 
-Each of the 4 buildbox components can be installed separately from their
-respective git repositiories, and each respository has individual install
-instructions. We recommend installing the latest release tag of each
-component.
-
-| **Buildbox-common:** See the installation section in:
-| https://gitlab.com/BuildGrid/buildbox/buildbox-common/-/blob/master/README.rst
-| (Be sure to install from the latest stable release tag.)
-
-| **Buildbox-casd:** See the installation section in:
-| https://gitlab.com/BuildGrid/buildbox/buildbox-casd/-/blob/master/README.rst \
-| (Be sure to install from the latest stable release tag.)
-
-| **Buildbox-fuse:** See
-| https://gitlab.com/BuildGrid/buildbox/buildbox-fuse/-/blob/master/INSTALL.rst
-| (Be sure to install from the latest stable release tag.)
-
-| **Buildbox-run-bublewrap:** See the installation section in:
-| https://gitlab.com/BuildGrid/buildbox/buildbox-run-bubblewrap/-/blob/master/README.rst
-| (Be sure to install from the latest stable release tag.)
+Buildbox components can be installed from their git
```

Comment:
This sounds like an incomplete sentence.

Reply:
Amended.